src/generators/modelsGenerator.ts

```diff
@@ -444,13 +444,17 @@ function writeUniontypes({ objects }: ClientDetails, modelsFile: SourceFile) {
     .forEach(obj => {
       const polymorphicObject = obj as PolymorphicObjectDetails;
       const childrenNames = [
-        obj.name,
         ...polymorphicObject.children.map(c => {
           return c.schema.children && c.schema.children.immediate.length
             ? `${c.name}Union`
             : c.name;
         })
       ];

```

Comment:
I think this could be better rewritten as:

```ts
const childrenNames = polymorphicObject.children.map(c =>
          c.schema.children && c.schema.children.immediate.length
            ? `${c.name}Union`
            : c.name
        );
```

Reply:
Good catch, by removing obj.name the spread is no longer needed.